.eslintrc.json

```diff
@@ -0,0 +1,28 @@
+{
+  "root": true,
+  "extends": [
+    "eslint:recommended",
+    "plugin:import/errors",
+    "plugin:import/warnings"
+  ],
+  "plugins": [
+    "react"
```

Comment:
I like that we are importing this, I do believe we should find a export preset of rules we can extend here to cover (most or all) of the rules we had previously set.

Reply:
Then what is the point of removing custom rules and creating dependency on other packages? The very last comment section of the issue #407 suggests a PR that removes custom rules, imports default, or am i wrong?